### ISSUE TITLE
Form field missing a label (Accessibility)

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/SearchWorks#3780]
Every form element should have a descriptive text label.

[SiteImprove Issue with details](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=8&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

This issue references several record view pages - specifically the browse related items section and the current item Select input. I looked at it and it doesn't look any different than the Select inputs for the related items. It might be a false positive, but needs to be verified by a developer.

**Comment by jcoyne** [comment]
<img width="581" alt="Screenshot 2024-01-24 at 6 48 59 AM" src="https://github.com/sul-dlss/SearchWorks/assets/92044/8f42be62-e4d7-4a11-b4b9-35973e1abdc4">
I think this is a false positive. It definitely has a label in the HTML.

**Comment by dbranchini** [comment]
Thank you for verifying, @jcoyne , I thought the same thing, but wanted to be sure. I'm going to close this as a false positive.
